### PR TITLE
Improve README documentation

### PR DESCRIPTION
**Pull request by seun-beta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,231 @@
...
+
+
+## 📝 License <a name = "license"></a>
+
+This project is licensed under the MIT License - see the [LICENSE](/LICENSE) file for more details.
+
+
+## 🎉 Acknowledgements <a name = "acknowledgement"></a>
+
+Special thanks to [Kosy](https://github.com/kosyfrances) for guiding us through the entire app creation process, reviewing our code and unblocking us when we are stuck with an issue, and most especially, for taking the time out of their busy lives to provide free mentorship to us
```
Comment (kosyfrances) [inline README.md]: Please add @toystars  as well

**Review by kosyfrances — COMMENTED** [review]
See comment

File: README.md [hunk 2]
```diff
@@ -1,15 +1,231 @@
...
+
+  ```sh
+  python3 -V
+  ```
+
+- Download [pip](https://pip.pypa.io/en/latest/installing/) and follow the instructions in the link as an installation guide.
+
+- [PostgreSQL](https://www.postgresql.org/download/windows/) (Ensure the server is running).
+
+- It is advisable to install Django in a virtual environment. The README uses [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html#basic-installation) to create this environment. You could use any virtualenv package of your choice but for Windows, install this wrapper with:
```
Comment (kosyfrances) [inline README.md]: Django? 😱

File: README.md [hunk 3]
```diff
@@ -1,15 +1,231 @@
...
+Add additional notes about how to deploy this on a live system.
+
+
+### Python installation instructions for Windows, macOS and other Linux distro Users
+
+- The following may serve as a guide:
+  - (https://www.python.org/downloads/)
+  - (https://realpython.com/installing-python/)
+  - (https://virtualenvwrapper.readthedocs.io/en/latest/install.html)
+  - (https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/development_environment)
```
Comment (kosyfrances) [inline README.md]: Django or flask?

File: README.md [hunk 4]
```diff
@@ -1,15 +1,231 @@
...
+- Clone the repository from the `staging` branch and please read the [contributing guide](/CONTRIBUTING.md).
+
+- You may also need to have [Heroku](https://devcenter.heroku.com/articles/heroku-cli).
+
+- Run the `Heroku` login commands in your terminal after installation
+  ```sh
+  heroku login
+  ```
+
+Contact [Kosy](https://github.com/kosyfrances), [Tosin](https://github.com/tosintubi) for more details.
```
Comment (kosyfrances) [inline README.md]: Not sure Kosy has info about the new Heroku app though 😅

File: README.md [hunk 5]
```diff
@@ -1,15 +1,231 @@
...
+  ```
+
+- Create a database with PostgresQL, if you installed it earlier. If not, installation instructions can be found [here](https://www.postgresql.org/download/windows/). Make sure to note database name, database username and password.
+
+
+
+
+## 🔧 Running the tests <a name = "tests"></a>
+
+Explain how to run the automated tests for this system.
```
Comment (kosyfrances) [inline README.md]: These are TODOs? right? Maybe add a note that it is a TODO?

File: README.md (hunk 2 above)
Comment (seun-beta) [reply README.md]: resolved

File: README.md (hunk 3 above)
Comment (seun-beta) [reply README.md]: resolved

**Comment by kosyfrances** [comment]
@seun-beta seems this PR does more than just improving the readme documentation. I'm not sure what the rest outside the readme file does.

**Comment by seun-beta** [comment]
> @seun-beta seems this PR does more than just improving the readme documentation. I'm not sure what the rest outside the readme file does.

It resolves issue #3

**Pull request closed without merging**